### PR TITLE
Add mongoDB Alats link.

### PR DESCRIPTION
**Pull request by Wenrh2004**

### CODE REVIEW

File: common/mongoConfig.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	"github.com/spf13/viper"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type MongoDB struct {
+	passWord    string
+	userName    string
+	databaseURL string
```
Comment (Wenrh2004) [inline common/mongoConfig.go:15]: Fix the MongoDB key leakage.

**Review by Wenrh2004 — COMMENTED** [review]
Add mongoDB server.

**Comment by ha0min** [comment]
When micro-services has interaction with DB, it should be initialized in repo/init.go, otherwise might not be able to initialized Gorm when running and get fatal error, see [f439385](https://github.com/Wenrh2004/Douyin_Demo/commit/f439385232ad0a95c622bd237d271adc642f885a)

**Comment by Wenrh2004** [comment]
> 

Gore isn't in favor of mongoDB,I used the official linking method given by mongoDB, I didn't look at microservices, I'll look at that later.

**Review by PolarishT — APPROVED** [review]
lgtm

**Pull request merged**